models/articles.js

```diff
@@ -18,3 +18,19 @@ exports.selectArticleByID = (id)=>{
     }))
 }
 
+exports.updateArticlesByArticleId = (id,increment) =>{
+    increment === undefined? increment = "hello": increment
```

Comment:
I would refactor to do a custom promise reject here to stop the execution before we get to the query when the increment value is undefined rather than set the vote value to a string